File: multiplesof3or5/dart/fransay.dart
// AUTHOR: Fransay
// DATE: 24/03/23
// DESCRIPTION: A program that returns the sum of multiples of 3 or 5 below 1000.

const numberLimit = 1000; 
void main(){
    var sum = 0;
    for (int i=0; i<numberLimit;i++){
        if (i%3 ==0 || i%5 == 0){
            sum+=i;
        }
    }
    print(sum);
}

File: multiplesof3or5/python/fransay.py
# AUTHOR: Fransay
# DATE: 24/93/23
# DESCRIPTION: A program that output the sum of the multiples of 3 or 5 less than 1000

LIMIT = 1000

def multiples_of_three(number):
    """returns a bool representation whether a number is three or not"""
    if number%3 == 0:
        return True 
    return False 

def multiples_of_five(number):
    """
    returns a bool representation whether a number is five or not
    """
    if number%5 == 0:
        return True
    return False


def main():
    """
    returns the sum of the multiples of 3 or 5 less than 1000
    """
    counter = 0
    sum = 0
    while counter < LIMIT:
        if multiples_of_three(counter) | multiples_of_five(counter):
            sum += counter
        counter+=1
    return sum


if __name__ == '__main__':
    print(main())
